scipy/stats/tests/test_qmc.py

```diff
@@ -559,18 +558,16 @@ def test_sample_stratified(self, optimization, centered,
         assert_allclose(sorted_sample, expected, atol=0.5 / n)
         assert np.any(sample - expected > 0.5 / n)
 
-    @pytest.mark.parametrize("centered", [False, True])
-    def test_orthogonal_array(self, centered):
-        seed = np.random.RandomState(123456)
-        p = 5
+        if strength == 2 and optimization is None:
+            p = d - 1
```

Comment:
This happens to be true for the values of `n` and `p` above, but I think the intent is
```suggestion
            p = sqrt(n)
```
You probably did this because `sqrt` would return a float, which can't be passed into `arange`. I think it is worth conversion to `int`, or better yet, define `p = 5` and the top and `n = p**2`.
Obviously has no bearing on the test; it's just to communicate the intent of the code.

Reply:
True, I went for defining p above.